Document.md

```diff
@@ -3,7 +3,7 @@
 The code section enclosed by `batch_start()` and `batch_flush()` is called batching segment. It can appear more than one time in a single application. Compared with typical syscalls, ESCA eliminates mode switches in batching segments by decoupling syscalls. Instead of switching to the kernel or executing the corresponding service routine, syscalls in batching segment only record their syscall ID and arguments in the shared table.After batch_flush is called, ESCA finally switches to kernel mode, executes all syscalls in the shared table, and then switches back to user mode.
 ## Typical system call flow
 ### Overview
-![](https://i.imgur.com/zEXqLKH.png)
+![image](assets/syscall-flow.png)
```

Comment:
This diagram IS NOT drawn by you, and you MUST mention the source and credit.

Reply:
@qwe661234 , as an alternative. You can use the figure I drawn in Page 6 in this [slide](https://docs.google.com/presentation/d/1aB-t-8wDnsQ07VTGmQd6F5bKZYseZlj8CJ80rJqsj_4/edit?usp=sharing).

Reply:
> @qwe661234 , as an alternative, you can use the figure I drawn in Page 6 in this [slide](https://docs.google.com/presentation/d/1aB-t-8wDnsQ07VTGmQd6F5bKZYseZlj8CJ80rJqsj_4/edit?usp=sharing).

Thanks, I changed image to the figure in Page 6 in this [slide](https://docs.google.com/presentation/d/1aB-t-8wDnsQ07VTGmQd6F5bKZYseZlj8CJ80rJqsj_4/edit#slide=id.p)